source/common/router/router.cc

```diff
@@ -562,6 +562,31 @@ Http::FilterHeadersStatus Filter::decodeHeaders(Http::RequestHeaderMap& headers,
     return Http::FilterHeadersStatus::StopIteration;
   }
 
+  // Support DROP_OVERLOAD config from control plane to drop certain percentage of traffic.
+  ENVOY_STREAM_LOG(debug, "Router filter: cluster DROP_OVERLOAD configuration: {}\n", *callbacks_,
```

Comment:
I suggest reducing this log line to a trace level, or otherwise it will be noisy.
Also what if there's no value set? Maybe this line should be inside the `if` scope below.

Reply:
Moved the log to inside if check. 
Could you please elaborate why it will crash if the value is not set?  We default set the value to be zero.  And if drop_overload_ is zero then the if check fail, and the drop overload logic is skipped.